ocs_ci/utility/lvmo_utils.py

```diff
@@ -126,13 +127,25 @@ def delete_lvm_cluster():
         execption if lvmcluster cant be deleted
     """
     clean_all_test_projects()
+    lvm_clustername = get_lvm_cluster_name()
     lmvcluster = OCP(kind="LVMCluster", namespace=constants.OPENSHIFT_STORAGE_NAMESPACE)
     try:
-        lmvcluster.delete(resource_name=constants.LVMCLUSTER)
+        lmvcluster.delete(resource_name=lvm_clustername)
     except CommandFailed as e:
-        if f'lvmclusters.lvm.topolvm.io "{constants.LVMCLUSTER}" not found' not in str(
-            e
-        ):
+        if f'lvmclusters.lvm.topolvm.io "{lvm_clustername}" not found' not in str(e):
             raise e
         else:
             log.info("LVMCluster not found, procced with creation of new one")
+
+
+def get_lvm_cluster_name():
+    """
+    Get LVM clustername Dynamically
+    returns:
+        (str) lvm cluster name.
+    """
+
+    if "lvms" in config.DEPLOYMENT["ocs_registry_image"]:
+        return "lvmscluster"
```

Comment:
Please change it to something in constants

Reply:
Done